customcommands/web.go

```diff
@@ -523,14 +525,54 @@ func handleUpdateCommand(w http.ResponseWriter, r *http.Request) (web.TemplateDa
 	dbModel.LocalID = cmdEdit.ID
 	dbModel.TriggerType = int(triggerTypeFromForm(cmdEdit.TriggerTypeForm))
 	// check low interval limits
-	if dbModel.TriggerType == int(CommandTriggerInterval) && dbModel.TimeTriggerInterval <= 10 {
-		if dbModel.TimeTriggerInterval < 5 {
-			dbModel.TimeTriggerInterval = 5
-		}
-
-		ok, err := checkIntervalLimits(ctx, activeGuild.ID, dbModel.LocalID, templateData)
-		if err != nil || !ok {
-			return templateData, err
+	if dbModel.TriggerType == int(CommandTriggerInterval) || dbModel.TriggerType == int(CommandTriggerCron) {
+		switch CommandTriggerType(dbModel.TriggerType) {
+		case CommandTriggerInterval:
+			if dbModel.TimeTriggerInterval <= 10 {
+				if dbModel.TimeTriggerInterval < 5 {
+					dbModel.TimeTriggerInterval = 5
+				}
+
+				ok, err := checkIntervalLimits(ctx, activeGuild.ID, dbModel.LocalID, templateData)
+				if err != nil || !ok {
+					return templateData, err
+				}
+			}
+		case CommandTriggerCron:
+			cronSpec := strings.TrimSpace(dbModel.TextTrigger)
```

Comment:
Perhaps it'd be more straightforward and/or reliable to reuse the parser from robfig/cron instead of ad-hoc string splitting. What do you think about something along the lines of (untested):
```go
sched, err := cron.NewParser(cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow).Parse(dbModel.TextTrigger)
if err != nil {
	return templateData, nil // invalid cron expression; should not get here
}
spec, ok := sched.(*cron.SpecSchedule)
if !ok {
	return templateData, nil // we don't enable constant delay schedules in parser; should not get here
}

var mins []int
for min := range 60 {
	if spec.Minute&(1<<min)) != 0 {
		mins = append(mins, min)
	}
}
if len(mins) == 0 {
	return templateData, nil
}

// ...check mins as usual?
```

Reply:
I got ChatGPT to explain this to me and still don’t understand it, so I’m not sure “straightforward” is the word I’d use. If Des agrees this is the best solution, please PR against my branch or wait until this PR is merged and open a new one. What’s here now has worked in all my testing, if you can provide an instance where it would not work that would be helpful.

Reply:
I've implemented your strategy instead of the string parsing. Thank you for the suggestion and snippet.